pkg/kubelet/dockershim/convert.go

```diff
@@ -169,3 +170,18 @@ func toRuntimeAPISandbox(c *dockertypes.Container) (*runtimeapi.PodSandbox, erro
 		Annotations: annotations,
 	}, nil
 }
+
+func checkpointToRuntimeAPISandbox(id string, checkpoint *PodSandboxCheckpoint) *runtimeapi.PodSandbox {
+	state := runtimeapi.PodSandboxState_SANDBOX_NOTREADY
+	labels := make(map[string]string)
+	labels[kuberuntime.KubernetesManagedLabel] = "true"
```

Comment:
Hmmmm. If so, why not checkpoint the complete PodSandboxConfig? I bet people will insist only checkpoint information that is needed.